src/Symfony/Component/Messenger/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+6.2
+---
+
+ * Throw an exception if no routing was found
```

Comment:
This is opt-in, the message should be adjusted to not suggest otherwise. (and the PR + commit titles also)